string robotID
aau_rng_multi_robot/PointM robotPosition
